packages/dd-trace/src/appsec/iast/path-line.js

```diff
@@ -13,11 +13,13 @@ const pathLine = {
 }
 
 const EXCLUDED_PATHS = [
-  path.join(path.sep, 'node_modules', 'diagnostics_channel')
+  path.join(path.sep, 'node_modules', 'dc-polyfill'),
+  path.join(path.sep, 'node_modules', 'dc-polyfill', 'dc-polyfill.js')
```

Comment:
Why did you include `path.join(path.sep, 'node_modules', 'dc-polyfill', 'dc-polyfill.js')`? It should not be necessary

```suggestion
  path.join(path.sep, 'node_modules', 'dc-polyfill')
```